src/Components/BookDetails.js

```diff
@@ -13,29 +15,42 @@ export default function BookDetails({ id, title, author }) {
 
   return (
     <li>
-      <div className="row">
+      <div className="row my-5 p-3 bg-white">
         <div className="col-md-6">
-          <h2>{title}</h2>
-          <h5>{author}</h5>
+          <p className="fs-6 mb-0 text-secondary">{category}</p>
+          <div><h2 className="fs-5 mb-0">{title}</h2></div>
+          <div><h5 className="mt-0 fs-6" style={{ color: '#4386bf' }}>{author}</h5></div>
           <span>
-            <button className="btn btn-primary btn-sm" type="button">
+            <button className="btn btn-default btn-sm text-primary" type="button">
               Comment
             </button>
             <button
-              className="btn btn-danger btn-sm"
+              className="btn btn-default text-primary btn-sm"
               type="button"
               onClick={handleRemoveBook}
             >
               Remove
             </button>
+            <button className="btn btn-default btn-sm text-primary" type="button">
+              Edit
+            </button>
           </span>
         </div>
-        <div className="col-md-6">
-          <div className="right-content">
-            <h3>Current Chapter</h3>
+        <div className="col-md-3 d-flex align-items-center">
+          <div className="rectangle">
+            <div className="inner-b" />
+          </div>
+          <div className="d-flex flex-column">
+            <p className="mb-0 fs-5">64%</p>
+            <small className="fs-6 text-muted">completed</small>
+          </div>
+        </div>
+        <div className="col-md-3">
+          <div>
+            <h3 className="fs-6 text-muted">Current Chapter</h3>
```

Comment:
Let's style each card to match the design as this is a requirement for this milestone😉😉.

design
<img width="813" alt="image" src="https://github.com/Innocent259/book-store/assets/64643995/d331cb7a-0f24-4697-a9a2-1195a13a4172">

yours
<img width="1354" alt="image" src="https://github.com/Innocent259/book-store/assets/64643995/3f6c775b-6611-44e5-858a-c675450fdaa4">

please take note of the font properties, spacings and ensure it matches the design😉.